src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/InterpreterLanguage.java

```diff
@@ -0,0 +1,326 @@
+    TUK_TUK("tuk-tuk", "Turkmen", ""),
+    TWI_TWI("twi-twi", "Twi", ""),
+    UIG_UIG("uig-uig", "Uighur", ""),
+    UKR_UKR("ukr-ukr", "Ukrainian", ""),
+    URD_URD("urd-urd", "Urdu", ""),
+    URH_URH("urh-urh", "Urohobo", ""),
+    UZB_UZB("uzb-uzb", "Uzbek", ""),
+    VIE_VIE("vie-vie", "Vietnamese", ""),
+    VSA_VSA("vsa-vsa", "Visayan", ""),
+    WEL_WEL("wel-wel", "Welsh", ""),
+    WOL_WOL("wol-wol", "Wolof", ""),
+    XHO_XHO("xho-xho", "Xhosa", ""),
+    YOR_YOR("yor-yor", "Yoruba", ""),
+    ZAG_ZAG("zag-zag", "Zaghawa", ""),
+    ZZA_ZZA("zza-zza", "Zaza", ""),
+    ZUL_ZUL("zul-zul", "Zulu", "");
+
+    private final String key;
+    private final String language;
+    private final String dialect;
+
+    public static InterpreterLanguage getInterpreterLanguageByLanguageAndDialect(String value, String dialect) {
+        for (InterpreterLanguage il : InterpreterLanguage.values()) {
+            if (il.getLanguage().equals(value) && il.getDialect().equals(dialect)) {
+                return il;
+            }
+        }
+        return null;
+    }
+
+    public static InterpreterLanguage getLanguageAndConvert(String languages){
+        return Arrays.stream(InterpreterLanguage.values())
+            .filter(c -> c.getLanguage().equals(languages))
+            .findFirst()
+            .orElse(conditionNotMet(languages));
+    }
+
+    public static InterpreterLanguage conditionNotMet(String languages){
+        switch (languages){
+            case "Arabic (Middle Eastern)":
+                return ARA_MIDDLEEASTERN;
+            case "Arabic (North African)":
+                return ARA_NORTHAFRICAN;
+            case "Ashanti":
+                return TWI_TWI;
+            case "Azari":
+                return AZE_AZE;
+            case "Bajuni":
+                return SWA_KIBAJUNI;
+            case "Bardini":
+                return KUR_KURMANJI;
+            case "Bengali (Sylheti)":
+                return BEN_SYLHETI;
+            case "Bharuchi":
+                return GUJ_GUJ;
+            case "Bravanese":
+                return SWA_BRAVANESE;
+            case "Cambellpuri":
+                return null;
+            case "Edo/Benin":
+                return BIN_BIN;
+            case "Emakhuna":
+                return null;
+            case "Feili":
+                return null;
+            case "French (Arabic)":
+                return FRE_ARABIC;
+            case "French(African)":
+                return FRE_AFRICAN;
+            case "Gallacian":
+                return null;
+            case "Guran":
+                return null;
+            case "Hendko":
+                return HND_HND;
+            case "Hokkien":
+                return null;
+            case "Ishan":
+                return null;
+            case "Karaninka":
+                return MNK_MNK;
+            case "Khymer Khymer":
+                return KHM_KHM;
+            case "Kibajuni,Kibanjuni,Bajuni,Ban":
```

Comment:
is this meant to be different cases? If so, separate them

Reply:
On the SSCS Data they are all under 1 String I thought the same at the time